src/screens/CreateNFT.js

```diff
@@ -0,0 +1,340 @@
+    const getAlertBody = () => {
+      return (
+        <div>
+          <p>{t`Your NFT has been successfully created!`}</p>
+          <p>{t`You can share the following configuration string with other people to let them register your brand new NFT in their wallets.`}</p>
+          <p><SpanFmt>{t`Remember to **make a backup** of this configuration string.`}</SpanFmt></p>
+          <p><strong>{this.state.configurationString}</strong></p>
+        </div>
+      )
+    }
+
+    const htrDeposit = hathorLib.tokens.getDepositPercentage() * 100;
+    const depositAmount = hathorLib.tokens.getDepositAmount(this.state.amount); 
+    const nftFee = hathorLib.helpers.prettyValue(tokens.getNFTFee());
+
+    return (
+      <div className="content-wrapper">
+        <BackButton {...this.props} />
+        <h3 className="mt-4">Create NFT</h3>
+        <p className="mt-5">{t`Here you will create a new NFT. After the creation, you will be able to send the units of this NFT to other addresses.`}</p>
+        <p>{t`NFTs share the address space with all other tokens, including HTR. This means that you can send and receive tokens using any valid address.`}</p>
+        <p>{t`Remember to make a backup of your new token's configuration string. You will need to send it to other people to allow them to use your NFT.`}</p>
+        <p>
+          {str2jsx(
+            t`When creating and minting NFTs, a |bold:deposit of ${htrDeposit}%| in HTR is required and an additional |bold:fee of ${nftFee} HTR|. If these tokens are later melted, this HTR deposit will be returned (depending on the amount melted) and the fee can never be returned. Read more about the NFT standard |link:here|.`,
+            {
+              bold: (x, i) => <strong key={i}>{x}</strong>,
+              link: (x, i) => <a key={i} href="true" onClick={this.goToRFC}>{x}</a>,
+            }
+          )}
+        </p>
+        <p>
+          {str2jsx(
+            t`The most common usage for an NFT is to represent a digital asset. Please see |link:this guide| that explains how to create an NFT data (including suggestions on how to upload files to the IPFS network) in the Hathor standard in order to be able to show your asset in our explorer.`,
+            {
+              link: (x, i) => <a key={i} href="true" onClick={this.goToNFTGuide}>{x}</a>,
+            }
+          )}
+        </p>
+        <hr className="mb-5 mt-5"/>
+        <form ref={this.formCreateNFTRef} id="formCreateNFT">
+          <div className="row">
+            <div className="form-group col-9">
+              <label>{t`NFT Data`}</label>
+              <input required ref={this.nftDataRef} placeholder={t`ipfs://...`} type="text" className="form-control" maxLength={NFT_DATA_MAX_SIZE} />
+              <small id="nftDataHelp" className="form-text text-muted">This can be the IPFS link to your metadata.json file, the link to your asset or a string that uniquely identify your NFT. Max size: {NFT_DATA_MAX_SIZE} characters.</small>
```

Comment:
Suggestion: You asset's URI (e.g. ipfs://YOUR_ASSET_ID).

Reply:
Are you suggesting to substitute the whole helper text by this or change a part of it?

`ipfs://...` is already in the placeholder, here I tried to describe what are the options.

Reply:
I was thinking about the whole helper. I think we should explain what should be put in this field and give the IPFS as an example. So, maybe "Put your asset's URI. For example, the IPFS link `ipfs://...`".

Reply:
This may cause a false impression that we support only URI as NFT data. This can actually be any string that represents the NFT. The `ipfs://...` is already in the placeholder as an example.

I think that maybe 99% of our NFTs might have a URI, so maybe that's enough for us to change to your suggestion but I'm not sure if we should.

Reply:
As we discussed in the call, I changed the nft helper message.